be/src/http/action/meta_action.cpp

```diff
@@ -60,13 +60,11 @@ Status MetaAction::_handle_header(HttpRequest* req, std::string* json_meta) {
         LOG(WARNING) << "no tablet for tablet_id:" << tablet_id << " schema hash:" << schema_hash;
         return Status::InternalError("no tablet exist");
     }
-    OLAPStatus s =
-            TabletMetaManager::get_json_meta(tablet->data_dir(), tablet_id, schema_hash, json_meta);
```

Comment:
Why are some rowset meta not in rocksdb but only in memory?
Is this part of rowset meta not effective?

Reply:
this problem is introduced by #1946 , tablet will do meta checkpoint periodly